tests/components/application-connector/resources/charts/gateway-test/templates/target-api-mock/mock-app.yml

```diff
@@ -2,31 +2,49 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels:
-    app: mock-application
-  name: mock-application
+    app: {{ .Values.mockServiceName}}
+  name: {{ .Values.mockServiceName}}
   namespace: {{.Values.namespace}}
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: mock-application
+      app: {{ .Values.mockServiceName}}
   template:
     metadata:
+      annotations:
+        traffic.sidecar.istio.io/includeInboundPorts: "*"
+        traffic.sidecar.istio.io/excludeInboundPorts: "8090"
       labels:
-        app: mock-application
+        app: {{ .Values.mockServiceName}}
     spec:
       containers:
         - image: {{ include "imageurl" (dict "reg" .Values.containerRegistry "img" .Values.images.mockApplication) }}
-          name: test-app
+          name: {{ .Values.mockServiceName}}
+          env:
+            - name: APP_CACERTPATH
+              value: "/etc/secret-volume/ca.crt"
+            - name: APP_SERVERCERTPATH
+              value: "/etc/secret-volume/server.crt"
+            - name: APP_SERVERKEYPATH
+              value: "/etc/secret-volume/server.key"
```

Comment:
Do we need these env variables? I think they are not used anywhere. These values are not configurable and are a part of the `NewConfig()` constructor